.travis.yml

```diff
@@ -36,6 +36,8 @@ env:
     - MINIO_ENDPOINT: http://localhost:9000/
     - MINIO_ACCESS_KEY_ID: example00000
     - MINIO_SECRET_ACCESS_KEY: secret00000
+    - secure: "px8XYeNEAFTSTb1hYZuEOxqOXUxvp3EoU+KCtPck/KNozkoS95eBd9klgr3Os4wPKloLdMhrr0VE98lukogUxA/NmnYnos01kegjWgwwM6fkob8JxaN5KK4oUFF1wmirBlrjGlw8vUErPwINmrK4BywKpDbw6Yip6FzxdlWESHI="
```

Comment:
Minio is not really being used. We can remove it and remove this secure line

Reply:
this secure is not for minio, it's for reviewdog. Will add comment